yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java

```diff
@@ -152,6 +158,43 @@ public String doLogin(@RequestParam("token") final String token, final Model mod
         return "profile";
     }
 
+    @RequestMapping("/dbs-standard")
```

Comment:
I updated the .NET example (in [this commit](https://github.com/getyoti/yoti-dotnet-sdk/pull/375/commits/e8092a1664dbe0f0c858263bf7315de4cfbbc61a)) to be `dbs-check` rather than `dbs-standard`, as I think it makes more sense from a client's perspective, they're just wanting to see a DBS check in action:
```suggestion
    @RequestMapping("/dbs-check")
```

Reply:
oh I see, sure no problem